docs/DEV.md

```diff
@@ -1,11 +1,11 @@
 # Developer Guide
 
-## Setup Development Environment
+## Setup the Development Environment
 
-### Without Docker
+### 1. Without Docker
 
 This is a _**TypeScript**_ project that uses _**React**_. You'll just need
-_[**Node.js**](https://nodejs.org/en) v16_ and _**npm**_ installed on your development machine.
+_[**Node.js**](https://nodejs.org/en) v16_ or higher and _**npm**_ installed on your development machine.
```

Comment:
We should confirm this assertion.

Reply:
@walterbender, I have tested it on v18 and v20 and it works fine on both.